package.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name": "chromatic-cli",
```

Comment:
I just realized that `chromatic-cli` has slipped into a few places here. For this release, we're not changing the name of the package so all name and references should remain `storybook-chromatic`

Reply:
@zol The package will be published using a publish script which will rename the package before publishing see:
https://github.com/chromaui/chromatic-cli/pull/1/files#diff-e6889d205eee7cf0d8b917c5576051c8R11

Reply:
which other places do you see that would still need changing?

Reply:
Ahh gotcha, didn't realize this was the case. That explains the other references to `chromatic-cli`